### PR TITLE
[de] Add German translation for "Install tools"

### PR DESCRIPTION
**Pull request by termdew**
This PR adds a German translation for "Install Tools".

When this PR gets merged, we can create a link  between **Getting started/Learning environment** and **Install Tools**, like in the English version.

### CODE REVIEW

File: content/de/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -3,3 +3,25 @@ title: "Werkzeuge installieren"
 weight: 10
 ---
 
+## kubectl
+
+Das Kubernetes Befehlszeilenprogramm [kubectl](/docs/user-guide/kubectl/) ermöglicht es Ihnen, Befehle auf einem Kubernetes-Cluster auszuführen. Sie können mit kubectl Anwendungen deployen, Cluster-Ressourcen überwachen und verwalten sowie Logs einsehen.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/_index.md:8]:
```suggestion
Das Kubernetes Befehlszeilenprogramm [`kubectl`](/docs/user-guide/kubectl/) ermöglicht es Ihnen, Befehle auf einem Kubernetes-Cluster auszuführen. Sie können mit `kubectl` Anwendungen bereitstellen, Cluster-Ressourcen überwachen und verwalten sowie Logs einsehen.
```

File: content/de/docs/tasks/tools/_index.md [hunk 2]
```diff
@@ -3,3 +3,25 @@ title: "Werkzeuge installieren"
...
+Weitere Informationen über alle verfügbaren kubectl-Befehle finden Sie in der [Kommandoreferenz von kubectl](/docs/reference/kubectl/).
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/_index.md:9]:
```suggestion
Weitere Informationen über alle verfügbaren `kubectl`-Befehle finden Sie in der [Kommandoreferenz von kubectl](/docs/reference/kubectl/).
```

File: content/de/docs/tasks/tools/_index.md [hunk 3]
```diff
@@ -3,3 +3,25 @@ title: "Werkzeuge installieren"
...
+
+kubectl kann unter Linux, macOS und Windows installiert werden. [Hier](install-kubectl) finden Sie Anleitungen zur Installation von kubectl.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/_index.md:11]:
```suggestion
`kubectl` kann unter Linux, macOS und Windows installiert werden. [Hier](install-kubectl) finden Sie Anleitungen zur Installation von kubectl.
```

File: content/de/docs/tasks/tools/_index.md [hunk 4]
```diff
@@ -3,3 +3,25 @@ title: "Werkzeuge installieren"
...
+
+## kind
+Mit [`kind`](https://kind.sigs.k8s.io/) können Sie Kubernetes lokal auf Ihrem Computer ausführen. Voraussetzung hierfür ist eine konfigurierte und funktionierende [Docker](https://docs.docker.com/get-docker/)-Installation.
+
+Die kind [Schnellstart](https://kind.sigs.k8s.io/docs/user/quick-start/)-Seite gibt Informationen darüber, was für den schnellen Einstieg mit kind benötigt wird.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/_index.md:16]:
```suggestion
Die `kind` [Schnellstart](https://kind.sigs.k8s.io/docs/user/quick-start/)-Seite gibt Informationen darüber, was für den schnellen Einstieg mit `kind` benötigt wird.
```

File: content/de/docs/tasks/tools/_index.md [hunk 5]
```diff
@@ -3,3 +3,25 @@ title: "Werkzeuge installieren"
...
+
+## minikube
+Ähnlich wie `kind` ist [`minikube`](https://minikube.sigs.k8s.io/) ein Tool, mit dem man Kubernetes lokal auf dem Computer ausführen kann. Minikube erstellt Cluster mit einer Node oder mehreren Nodes. Somit ist es ein praktisches Tool für tägliche Entwicklungsaktivitäten mit Kubernetes, oder um Kubernetes einfach einmal lokal auszuprobieren. 
+
+[Hier](/install-minikube) erfahren Sie, wie Sie Minikube auf Ihrem Computer installieren können.
+Falls Sie minikube bereits installiert haben, können Sie es verwenden, um eine [Beispiel-Anwendung zu deployen](/docs/tutorials/hello-minikube/).
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/_index.md:22]:
```suggestion
Falls Sie `minikube` bereits installiert haben, können Sie es verwenden, um eine [Beispiel-Anwendung zu deployen](/docs/tutorials/hello-minikube/).
```

File: content/de/docs/tasks/tools/_index.md [hunk 6]
```diff
@@ -3,3 +3,25 @@ title: "Werkzeuge installieren"
...
+
+## kubeadm
+Mit kubeadm können Sie Kubernetes-Cluster erstellen und verwalten. kubeadm führt alle notwendigen Schritte aus, um ein minimales aber sicheres Cluster in einer benutzerfreundlichen Art und Weise aufzusetzen.
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/_index.md:25]:
```suggestion
Mit `kubeadm` können Sie Kubernetes-Cluster erstellen und verwalten. `kubeadm` führt alle notwendigen Schritte aus, um ein minimales aber sicheres Cluster in einer benutzerfreundlichen Art und Weise aufzusetzen.
```

**Comment by termdew** [comment]
Thank you @bene2k1 for the feedback! I applied the suggested changes.

**Pull request merged**